packages/google_atlas/pubspec.yaml

```
@@ -16,8 +16,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  atlas: 
-    path: "../atlas"
+  atlas: ^0.5.0
```

Can we set this to the latest version?
```suggestion
  atlas: ^0.5.1
```